plugins/kyverno/main.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+set -e
+trap 'echo "Error on Line: $LINENO"' ERR
+echo "Starting kyverno"
+results_file=/output/kyverno.json
+
+json='{"policyReports":[], "clusterPolicyReports":[]}'
+
+KIND="policyreport"
+
+echo "Retrieving namespaces"
+namespaces=$(kubectl get namespaces -o name)
+IFS=$'\n' namespaces=($namespaces)
+for ns_idx in "${!namespaces[@]}"; do
```

Comment:
I think there's a way to just iterate over values
```
for value in "${fruits[@]}"
do
  echo "Value for fruits array is: $value"
done
```

Reply:
Ah yeah that's cleaner. I cribbed this part from the kubesec script.